app/src/main/java/com/example/lhi06/residue/Album.java

```diff
@@ -29,15 +29,18 @@ public String getArtworkUrl() {
         return artworkUrl100;
     }
 
-    public void setArtworkUrl(@NonNull final String artworkUrl100) {
-         this.artworkUrl100 = artworkUrl100;
+    @NonNull
+    public void setArtworkUrl(String artworkUrl100) {
```

Comment:
revert this as it was correct before - function is void so does not return anything and so @NonNull not applicable for function return but is appropriate for artworkUrl100 parameter (as well as 'final')